mnefun/_mnefun.py

```diff
@@ -949,7 +949,7 @@ def run_sss_locally(p, subjects, run_indices):
             raw.fix_mag_coil_types()
             raw = filter_chpi(raw)
 
-            assert isinstance(p.trans_to, string_types)
+            assert type(p.trans_to) in (str, np.ndarray, type(None))
```

Comment:
`assert` statements can be skipped by Python when it's executed with `-O`. What you probably want in any case is `isinstance(p.trans_to, (string_types, np.ndarray, type(None)))`